app/scripts/templates/settings.mustache

```diff
@@ -2,7 +2,7 @@
   <header id="fxa-settings-header">
     <h1 id="fxa-manage-account">{{#t}}Firefox Accounts{{/t}}</h1>
     {{#showSignOut}}
-      <button id="signout" class="settings-button secondary">{{#t}}Sign out{{/t}}</button>
+      <button id="signout" class="settings-button secondary-button">{{#t}}Sign out{{/t}}</button>
```

Comment:
On smaller mobile screens look liked the button moved below the header.

<img width="397" alt="screen shot 2018-05-03 at 11 07 48 am" src="https://user-images.githubusercontent.com/1295288/39585255-54e13026-4ec2-11e8-9dea-c2cae78a1f12.png">

Reply:
Fixed.